File: tasks/books/hanbit/test-for-job/7/3/1.py
def slice_rc(arr: list, h: int, m: int):
    margin = 0
    while m > margin:
        h -= 1
        margin = 0
        for rc in arr:
            if rc > h:
                margin += rc - h
    return h

if __name__ == '__main__':
    n, m = map(int, input().split())
    arr = list(map(int, input().split()))
    print(slice_rc(arr, max(arr), m))

File: tasks/lectures/jnu/c-ce-basic-kim-2022-1/l-5.30-pointer-array/1.c
#include <stdio.h>

int main () {
  int x[3] = { 1, 2, 3 };
  int y[3] = { 4, 5, 6 };
  int z[3] = { 7, 8, 9 };
  int* arr[3] = { x, y, z };
  int i;

  for (i = 0; i < 3; i++) {
    for (int j = 0; j < 3; j++) {
      printf("%d ", arr[i][j]);
    }
    printf("\n");
  }
}